app/routes/index.js

```diff
@@ -1,40 +1,34 @@
 'use strict';
 
-// Libs
-const express = require('express');
-const logger = require('winston');
-
-// App modules
-const stathat = require('../../lib/stathat');
-
-// Router
-const router = express.Router(); // eslint-disable-line new-cap
-
-router.get('/', (req, res) => {
-  stathat.postStat('route: /');
-  return res.send('hi');
-});
-
-/**
- * Authentication.
- * TODO: Take out to separate middleware
- */
-router.use((req, res, next) => {
-  const apiKey = process.env.GAMBIT_API_KEY;
-  if (req.method === 'POST' && req.get('x-gambit-api-key') !== apiKey) {
-    stathat.postStat('error: invalid x-gambit-api-key');
-    logger.warn('router invalid x-gambit-api-key:', req.url);
-
-    return res.sendStatus(403);
-  }
-  return next();
-});
-
-router.use('/v1/status', (req, res) => res.send('ok'));
-router.use('/v1/campaigns', require('./campaigns'));
-router.use('/v1/chatbot', require('./chatbot'));
-router.use('/v1/signups', require('./signups'));
+// Routes
+const campaignsRoute = require('./campaigns');
+const chatbotRoute = require('./chatbot');
+const signupsRoute = require('./signups');
+const statusRoute = require('./status');
+const homeRoute = require('./home');
+
+// middleware config
+const authenticateConfig = require('../../config/middleware/authenticate');
+
+// middleware
+const timeoutMiddleware = require('../../lib/middleware/timeouts');
+const authenticateMw = require('../../lib/middleware/authenticate');
```

Comment:
This is minor and nitpicky, but what do you think about defining this as  `authenticateMiddleware` to stay consistent with other constant names? My personal pref would be to consistently spell them out as `Middleware` instead of `Mw` for readability.